src/bunyanstream.js

```diff
@@ -0,0 +1,42 @@
+import { Writable } from 'stream';
+import _ from 'lodash';
+import * as defaults from './defaults';
+import Logger from './logger';
+
+/**
+ * BunyanStream class.
+ */
+class BunyanStream extends Writable {
+  constructor(opts) {
+    super({
+      objectMode: true
+    });
+
+    const loggerOpts = _.clone(opts || {});
```

Comment:
I would rather use "Object.assign" than a third party library to clone an object. ;)